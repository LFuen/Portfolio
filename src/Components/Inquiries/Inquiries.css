.neumorphic {
    box-shadow: 12px 12px 24px 0 rgba(0, 0, 0, 0.2),
        -12px -12px 24px 0 rgba(255, 255, 255, 0.5);
}

.neumorphic--inset {
    box-shadow: inset 6px 6px 10px 0 rgba(0, 0, 0, 0.2),
        inset -6px -6px 10px 0 rgba(255, 255, 255, 0.5);
}

.pressed {
    justify-content: space-around;
    align-items: center;
}