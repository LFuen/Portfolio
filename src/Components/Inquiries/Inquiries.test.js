import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Inquiries from './Inquiries'


describe(`Inquiries component`, () => {
    describe(`Snapshot`, () => {
        it(`renders the UI as expected`, () => {
            const inquiries = renderer.create(
                <BrowserRouter>
                    <Inquiries />
                </BrowserRouter>).toJSON()
            expect(inquiries).toMatchSnapshot()
        })
    })

    describe(`Smoke test`, () => {
        it(`renders without crashing`, () => {
            const div = document.createElement('div')

            ReactDOM.render(
                <BrowserRouter>
                    <Inquiries />
                </BrowserRouter>,
                div
            )
            ReactDOM.unmountComponentAtNode(div)
        })
    })
})