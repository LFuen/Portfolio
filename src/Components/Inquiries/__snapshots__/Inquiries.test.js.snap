// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inquiries component Snapshot renders the UI as expected 1`] = `
<div
  className="App"
>
  <article
    className="margin"
  >
    <h1
      id="Inquiries"
    >
      Inquiries
    </h1>
    <div
      className="bg-color"
    >
      <form
        action="https://formspree.io/lile7886@gmail.com"
        method="post"
      >
        <fieldset>
          <legend>
            E-Mail Me
          </legend>
          <label
            htmlFor="user-name"
          >
            Name:
          </label>
          <input
            className="neumorphic neumorphic--inset pressed"
            id="user-name"
            name="name"
            placeholder="Name"
            type="text"
          />
          <label
            htmlFor="email"
          >
            E-Mail:
          </label>
          <input
            className="neumorphic neumorphic--inset pressed"
            id="email"
            name="email"
            placeholder="E-Mail"
            required={true}
            type="text"
          />
          <label
            htmlFor="email-message"
          >
            Questions? Comments? Requests?
          </label>
          <textarea
            className="neumorphic neumorphic--inset pressed"
            cols="30"
            id="email-message"
            name="message"
            placeholder="How may I help you?"
            rows="10"
          />
          <button
            type="submit"
          >
            Submit
          </button>
        </fieldset>
      </form>
    </div>
  </article>
</div>
`;
